From 9e8e2f5fd87a2df767639e29d2cd3de968a6639f Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 24 Mar 2023 09:26:57 +0100
Subject: [PATCH 0319/1695] plat-stm32mp2: conf: support BSEC3 driver

Default enable BSEC3 driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git dcf645eec216710daf83cc3b1f47d161b153ac5b plat-stm32mp2: conf: support BSEC3 driver]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ie46e6a36bc851390b228de478fefa6ccba1ee33b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/247731
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 09fa843fc..ebee31883 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -66,6 +66,7 @@ CFG_NUM_THREADS ?= 5
 CFG_TEE_CORE_NB_CORE ?= 2
 CFG_STM32MP_OPP_COUNT ?= 3
 
+CFG_STM32_BSEC3 ?= y
 CFG_STM32_GPIO ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_RIF ?= y
-- 
2.47.2

